pyproject.toml

```diff
@@ -11,6 +11,9 @@ keywords = ["packaging", "dependency", "poetry"]
 classifiers = [
     "Topic :: Software Development :: Build Tools",
     "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Software Development :: Libraries :: Python Modules",
```

Comment:
Duplicated classifier

Reply:
Good catch, I wondered why it was marked as changed but I missed the reason.